pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>11</jdk.version>
+        <jdk.version>8</jdk.version>
```

Comment:
return the original version

Reply:
with original version my program doesnt work,i tried different solutions of this problem

Reply:
you should install 11 version and then it will work, it's better to make this for the next projects, so you don't need to change the version in pom.xml every time